dash_textarea_autocomplete/package-info.json

```diff
@@ -15,8 +15,13 @@
     "build:js": "webpack --mode production",
     "build:backends": "dash-generate-components ./src/lib/components dash_textarea_autocomplete -p package-info.json --r-prefix '' --jl-prefix ''",
     "build:backends-activated": "(. venv/bin/activate || venv\\scripts\\activate && npm run build:py_and_r)",
-    "build": "npm run build:js && npm run build:backends",
-    "build:activated": "npm run build:js && npm run build:backends-activated"
+    "build:create-tar-file-from-deps": "tar -czf ./deps.tar.gz ./deps && mv ./deps.tar.gz ./deps",
+    "build": "npm run build:js && npm run build:backends && npm run build:create-tar-file-from-deps",
+    "build:activated": "npm run build:js && npm run build:backends-activated",
+    "postbuild:fixup-external-url-jl": "sed -i 's/https:\\/\\/unpkg.com\\/dash_textarea_autocomplete/https:\\/\\/unpkg.com\\/dash-textarea-autocomplete/' src/DashTextareaAutocomplete.jl",
+    "postbuild:fixup-dash-version-project-jl": "sed -i 's/Dash = \"0.1.3\"/Dash = \"0.1.3, 1.0\"/' Project.toml",
+    "postbuild:fixup-resources-path-jl": "sed -i 's/realpath(joinpath( @__DIR__, \"..\", \"deps\"))/artifact\"dash_textarea_autocomplete_resources\"/' src/DashTextareaAutocomplete.jl",
+    "postbuild": "npm run postbuild:fixup-external-url-jl && npm run postbuild:fixup-dash-version-project-jl && npm run postbuild:fixup-resources-path-jl"
```

Comment:
This is a _lot_ of manual fix-ups, it would be nice to follow up on creating PR in the `dash` repo so we can remove each of these one by one. We can create Issues here to track it potentially ?

Reply:
Good point!

I would start by opening an issue in https://github.com/plotly/dash/issues mentioning that the "proper" way (and the one that's work with `PackageCompiler.jl` :smirk: ) to handle artifacts in Julia now is by defining `Artifact.toml`.

Then, we should mention that `Artifact.toml` only accepts tarball, so we would need to tar the `min.js` and `min.js.map` files together in the build process so that the tarball is part of the npm registry and has a permanent URL available from unpkg.com

We could also reference this PR (they can see it, it's opensource :confetti_ball: ) to put things in context.